webapp/src/main/webapp/src/app/sender/sync/sender-sync-message.component.html

```diff
@@ -27,8 +28,9 @@ <h5><span class="badge badge-dark" i18n="@@common-total-count">Total Count</span
 			<i class="bi-envelope-dash text-warning ui-cursor-pointer" [hidden]="m.status != 'NEW'" [ngbTooltip]="statusWaitingTemplate" [openDelay]="1000"></i>
 			<i class="bi-envelope-check-fill text-success ui-cursor-pointer" [hidden]="m.status != 'SENT'" [ngbTooltip]="statusSentTemplate" [openDelay]="1000"></i>
 		</td>
-		<td>{{m.dateCreated | date:'yyyy-MM-dd HH:mm:ss'}}</td>
-		<td>{{m.dateSent | date:'yyyy-MM-dd HH:mm:ss'}}</td>
+		<td container="body" ngbTooltip="{{m.eventDate | date:'yyyy-MM-dd HH:mm:ss'}}" [openDelay]="1000">{{m.eventDate | date:'yyyy-MM-dd'}}</td>
```

Comment:
Why are you adding container attribute? It wasn't there before

Reply:
I thought on the call we said for the tooltip we should consider using a more user readable date format, in fact even the data we should use a more friendly format, so for the date use angular's predefined formats **mediumDate** format and for the tooltip use **full** format, let me know what you think when you try these formats, I think they look cooler.

Reply:
Done